explorer/ui/app/routes/xblock.$id/route.tsx

```diff
@@ -0,0 +1,28 @@
+import { Footer } from '~/components/shared/footer'
+import Navbar from '~/components/shared/navbar'
+import { GetXBlock } from '../../components/queries/block'
+import { useParams, useSearchParams } from '@remix-run/react'
+
+export default function Index() {
+  const params = useParams()
+  const [searchParams] = useSearchParams()
+
+  const data = GetXBlock(params.id || '', searchParams.get('height') || '')
```

Comment:
This is a wraps `useQuery` hook - so itself is a hook. should follow `use` naming convention yeah? `useXBlock`?

Also - we should can do this in another PR. But would like to make sure all big page queries are rendered on server before rendering the page. Right now the ui doesn't do that. 

Would be nice if we could use in-component queries and that would just work - but it seems it doesn't?

Let's use remix [loaders](https://remix.run/docs/en/main/route/loader)- and [useLoaderData](https://remix.run/docs/en/main/hooks/use-loader-data)